app/views/hiring_staff/vacancies/review.html.haml

```diff
@@ -8,229 +8,28 @@
     %p.govuk-body-l
       - if @vacancy.publish_on.today?
         = t('jobs.review')
-      - else 
+      - else
         = t('jobs.review_future')
     = render 'hiring_staff/vacancies/error_messages', errors: @vacancy.errors
 
 .vacancy.govuk-grid-row
   .govuk-grid-column-two-thirds
+    = render 'hiring_staff/vacancies/review_vacancy_sections/review_job_specification'
+    - if UploadDocumentsFeature.enabled?
+      = render 'hiring_staff/vacancies/review_vacancy_sections/review_supporting_documents'
+    - else
```

Comment:
After the feature flag is on, there may still be jobs with candidate specification data that haven't yet been published - correct?

Reply:
Apologies, this is out of scope for the PR - rebased and re-pushed to reflect this.